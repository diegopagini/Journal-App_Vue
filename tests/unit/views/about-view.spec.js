/** @format */
import AboutView from '@/views/AboutView.vue';
import { shallowMount } from '@vue/test-utils';

describe('AboutView', () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallowMount(AboutView);
	});

	it('Should render the component', () => {
		expect(wrapper.html()).toMatchSnapshot();
	});
});
