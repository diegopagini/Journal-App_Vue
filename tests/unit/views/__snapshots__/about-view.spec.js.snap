// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView Should render the component 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
